x/interchainqueries/keeper/keeper_test.go

```diff
@@ -566,12 +567,22 @@ func (suite *KeeperTestSuite) TestQueryTransactions() {
 	}
 	queriesKeeper.SetLastSubmittedTransactionIDForQuery(ctx, queryID, lastID)
 
-	start, end = 3, 8
+	start = 3
```

Comment:
Please set a limit here, too, just for clarity
```suggestion
	start, limit = 3, 5
```